core/src/sigverify_shreds.rs

```diff
@@ -121,7 +136,7 @@ fn verify_packets(
             .filter_map(|(slot, pubkey)| Some((slot, pubkey?.to_bytes())))
             .chain(std::iter::once((Slot::MAX, [0u8; 32])))
             .collect();
-    let out = verify_shreds_gpu(packets, &leader_slots, recycler_cache);
+    let out = verify_shreds_gpu(thread_pool, packets, &leader_slots, recycler_cache);
```

Comment:
Passing CPU thread pool into `verify_shreds_gpu` .. barf.

By the way, not blaming you for the precedent that was already here.